example/static/js/csrf_token.js

```diff
@@ -1,32 +0,0 @@
-/**
```

Comment:
Django teraz place ze nema example/static adresar. Bud vyrob prazdny adresar so suborom .gitkeep, alebo skus zistit ako spravit settings tak aby to nepotreboval.

Problem sa objavi az ked sa pokusis naladovat stranku.